electroncash/address.py

```diff
@@ -88,10 +88,14 @@ def double_sha256(x):
     return sha256(sha256(x))
 
 def ripemd160(x):
-    '''Simple wrapper of hashlib ripemd160.'''
-    h = _new_hash('ripemd160')
-    h.update(x)
-    return h.digest()
+    try:
+        md = hashlib.new('ripemd160')
+        md.update(x)
+        return md.digest()
+    except:
+        from . import ripemd
```

Comment:
Just to document why we are doing this

```suggestion
        # Fall-back to native python (in case hashlib lacks native ripemd160)
        from . import ripemd
```